docs/new_assignment.rst

```diff
@@ -0,0 +1,57 @@
+Creating a new assignment template
+----------------------------------
+
+Creating a new assignment template involves getting your local assignment
+files into
+a new repository on GitHub that you can use as a template for a GitHub
+classroom assignment.
+
+To create a new assignment template called "assignment1" using abc-classroom::
+
+  $ abc-assignment_template -a assignment1
```

Comment:
i tried to reinstall it and still can't see that -a flag even when i call the help... any suggestions? i'll keep troubleshooting.

Reply:
this is what i'm seeing

assignment_template doesn't work
assignment-template does but -a isn't there

```
(abc-dev) cu-biot-2-10:abc-classroom leahwasser$ abc-assignment_template -a
-bash: abc-assignment_template: command not found
(abc-dev) cu-biot-2-10:abc-classroom leahwasser$ abc-assignment_template -a sdfs
-bash: abc-assignment_template: command not found
(abc-dev) cu-biot-2-10:abc-classroom leahwasser$ abc-assignment-template -a sdfs
usage: abc-assignment-template [-h] [--custom-message] [--local-only]
                               [--mode {delete,fail,merge}]
                               assignment

```

Reply:
this is in setup.py
i like the consistent dashes better than remembering dashes vs underscores. 

`"abc-assignment-template = abcclassroom.__main__:assignment_template",`